reactjs/react2/week1/new-app/app/nasaepic/epic/page.js

```diff
@@ -0,0 +1,52 @@
+
+"use client"
+import { useRouter } from "next/navigation";
+import { useEffect, useState } from "react";
+import Navbar from "@/components/Navbar";
+
+const NASA_EPIC_API_KEY = process.env.NEXT_PUBLIC_API_KEY;
+
+const EpicImage = () => {
+  const router = useRouter();
+  const [date, setDate] = useState("");
+  const [imageUrl, setImageUrl] = useState("");
+
+  useEffect(() => {
+    if (router.isReady) {
```

Comment:
Where did you get this API? 
This is what `console.log(router)` shows, there is no `isReady` in there.
 
<img width="286" alt="image" src="https://github.com/user-attachments/assets/3e238377-21de-4a70-ae2a-fc82318a501e">

Also it was expected to use `useSearchParams` hook here, because `router.query` also returns `undefined` to me 🤔

Reply:
I have fixed this @markitosha